app/views/Export/ExportCodeInput.js

```diff
@@ -88,22 +90,20 @@ const CodeInput = ({ code, length, setCode }) => {
 
   return (
     <View style={{ flexDirection: 'row', flexShrink: 1 }}>
-      {[...digits].map((digit, i) => (
+      {[...characters].map((character, i) => (
```

Comment:
Why the array clone? .map will not mutate the original array.

Reply:
good catch